### ISSUE TITLE
Startup of Docker Image fails

### ISSUE THREAD

**Issue by rhuss** [octobox/octobox#839]
Are you experiencing this issue in octobox.io or your own instance?
 - [ ] octobox.io
 - [X] My instance

🐞 **Problem**

Latest image version from yesterday Octobox can't start anymore on OpenShift. Not sure whether this is an issue or the update https://github.com/octobox/octobox/commit/3b2c9852bcfe7fc2379dd803bec1f8a4cb7269f9 which causes this issue.

```
`/` is not writable.
Bundler will use `/tmp/bundler/home/unknown' as your home directory temporarily.
rake aborted!
LoadError: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? Error relocating /usr/local/bundle/gems/pg-1.1.0/lib/pg_ext.so: rb_enc_alias: symbol not found - /usr/local/bundle/gems/pg-1.1.0/lib/pg_ext.so
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/pg-1.1.0/lib/pg.rb:4:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/postgresql_adapter.rb:5:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/connection_specification.rb:191:in `spec'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:950:in `establish_connection'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_handling.rb:60:in `establish_connection'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/railtie.rb:136:in `block (2 levels) in <class:Railtie>'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:71:in `instance_eval'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:71:in `block in execute_hook'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:43:in `block in on_load'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:42:in `each'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:42:in `on_load'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/railtie.rb:132:in `block in <class:Railtie>'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:361:in `initialize!'
/usr/src/app/config/environment.rb:5:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:337:in `require_environment!'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:520:in `block in run_tasks_blocks'
/usr/local/bundle/gems/bugsnag-6.8.0/lib/bugsnag/integrations/rake.rb:18:in `execute_with_bugsnag'
/usr/local/bundle/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'

Caused by:
LoadError: Error relocating /usr/local/bundle/gems/pg-1.1.0/lib/pg_ext.so: rb_enc_alias: symbol not found - /usr/local/bundle/gems/pg-1.1.0/lib/pg_ext.so
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/pg-1.1.0/lib/pg.rb:4:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/postgresql_adapter.rb:5:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/connection_specification.rb:191:in `spec'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:950:in `establish_connection'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/connection_handling.rb:60:in `establish_connection'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/railtie.rb:136:in `block (2 levels) in <class:Railtie>'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:71:in `instance_eval'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:71:in `block in execute_hook'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:43:in `block in on_load'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:42:in `each'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:42:in `on_load'
/usr/local/bundle/gems/activerecord-5.2.1/lib/active_record/railtie.rb:132:in `block in <class:Railtie>'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:361:in `initialize!'
/usr/src/app/config/environment.rb:5:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:337:in `require_environment!'
/usr/local/bundle/gems/railties-5.2.1/lib/rails/application.rb:520:in `block in run_tasks_blocks'
/usr/local/bundle/gems/bugsnag-6.8.0/lib/bugsnag/integrations/rake.rb:18:in `execute_with_bugsnag'
/usr/local/bundle/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'
Tasks: TOP => db:migrate => db:load_config => environment
(See full trace by running task with --trace)
```

**Comment by rhuss** [comment]
It about this image: octoboxio/octobox@sha256:0e965fca226b9f64846277029015b8c3c8f63fd8da32b3c81dbacc6830d023a

**Comment by rhuss** [comment]
Btw, this happens also when running with `docker-compose`, so for me it looks like the image is broken.

**Comment by thiesschneider** [comment]
I have the same issue :/

**Comment by andrew** [comment]
I suspect it was https://github.com/octobox/octobox/pull/837, reverting here: https://github.com/octobox/octobox/pull/840

**Comment by andrew** [comment]
Can anyone see if this is still broken on the latest build?

**Comment by andrew** [comment]
I think this bug is specific to alpine, looks like there's been a patch released in 1.1.1, https://github.com/ged/ruby-pg/pull/28, but the changelog doesn't appear to have been updated, going to test it locally before updating.

**Comment by andrew** [comment]
Yeah appears still broken in 1.1.1, going to lock it to 1.0.0 for now.
